src/pages/productPage.ts

```diff
@@ -103,4 +111,14 @@ export default class ProductPage extends BasePage {
       await expect(this.productPageLocators.productCountInCompareListText).toHaveText(expectedRegex, { timeout: 15000 });
     });
   }
+
+  async verifyProductInWishlistByName(productName: string) {
+    await test.step(`Verify product "${productName}" is added to Wishlist`, async () => {
+      // Локатор для повідомлення про успішне додавання до списку бажаного
+      const successMessageLocator = this.page.locator('div.message-success.success.message');
+
+      // Очікуємо, що повідомлення буде містити правильний текст
+      await expect(successMessageLocator).toContainText(`${productName} has been added to your Wish List.`);
```

Comment:
If you expect strict text match for element  content use toHaveText() assertion function. The better way for use toContainText()  when you expect some substring inside element content.